gettsim/transfers/rente.py

```diff
@@ -296,3 +315,62 @@ def ges_rente_regelaltersgrenze(geburtsjahr: int, ges_rente_params: dict) -> flo
         ],
     )
     return out
+
+
+def ges_rente_grenze_altersrente(
```

Comment:
Is `grenze_altersrente` the correct term? I didn't find it directly when googling it. Also the term sounds quite similar to `regelaltersgrenze`.

Maybe `grenze_vorgez_altersrente` is more intuitive?

Reply:
I did not find an official term for this. "Altersrente" is the official term. In the literature this is often called the "full retirement age" maybe '''grenze_volle_altersrente '''?

Reply:
changed to latter

Reply:
ah! I thought you made this up. sorry.

Reply:
I should have added the source!

I am not completely sure if it is the same concept. But if it is I think it is the more intuitive term.

Reply:
actually the souce shows something else for me...

but I think you are right. The vorgezogene altersrente includes also early retirment with deductions. What I want is to assign an age threshold for the first retirement without deductions.